JHUGenerator/main.F90

```diff
@@ -1,7 +1,9 @@
 ! If you use this program please cite Phys.Rev. D81 (2010) 075022; arXiv:1001.3396  [hep-ph],
 !                                     Phys.Rev. D86 (2012) 095031; arXiv:1208.4018  [hep-ph],
 !                                     Phys.Rev. D89 (2014) 035007; arXiv:1309.4819  [hep-ph],
-!                                 and Phys.Rev. D94 (2016) 055023; arXiv:1606.03107 [hep-ph].
+!                                     Phys.Rev. D94 (2016) 055023; arXiv:1606.03107 [hep-ph],
```

Comment:
Please remove all these lines in main.F90.

Reply:
What is wrong with CircleCi?
No new lines are introduced, so nothing to remove.

Reply:
Then please remove this change since it is unnecessary.

Are all Ci checks reestablished? If so, they are bound to fail since MELA is not updated. Please make the corresponding PR.

Reply:
Can we finally move forward and release this version?